### PR TITLE
Break calicoctl into submodules

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

**Comment by djosborne** [comment]
Planning on making two follow up pull requests for
- Distributed param parsing
- UTs
I will follow up to make sure those tasks are started by the end of the week

File: calico_containers/calico_ctl/pool.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Usage:
+  calicoctl pool (add|remove) <CIDR> [--ipip] [--nat-outgoing]
+  calicoctl pool show [--ipv4 | --ipv6]
+
+Description:
+  Configure IP Pools
```
Comment (djosborne) [inline calico_containers/calico_ctl/pool.py]: Got any suggestions?

Comment (tomdee) [inline calico_containers/calico_ctl/pool.py]: Nope.

**Comment by tomdee** [comment]
LGTM

**Pull request merged**